ui/src/features/common/DetailsBox.tsx

```
@@ -86,38 +85,28 @@
-                      value: e.currentTarget.value,
-                    })
-                  }
-                />
-              </div>
-            ) : (
-              <div className="flex items-center gap-1" key={detail.id}>
-                <p>{detail.name}</p>
+          <>
+            <div className="flex flex-col gap-2">
+              {filteredContent.map(({ name }) => (
+                <p>{name}</p>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)